lib/vim-plugin.nix

```
@@ -1,5 +1,34 @@
 { lib }:
+let
+  settingsOptionDescription =
```

```suggestion
  mkSettingsOptionDescription =
```

I think a `mk` prefix is appropriate for factory functions